test/e2e/specs/site-editor/style-book.spec.js

```diff
@@ -42,7 +42,7 @@ test.describe( 'Style Book', () => {
 		).not.toBeVisible();
 		await expect(
 			page.locator( 'role=button[name="View"i]' )
-		).not.toBeVisible();
+		).toHaveAttribute( 'aria-disabled', 'true' );
```

Comment:
We should use `.toBeDisabled()` here. It checks both `disabled` and `aria-disabled`.

Ref: https://playwright.dev/docs/api/class-locatorassertions#locator-assertions-to-be-disabled.